### PR TITLE
Sq handlers

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: handlers/shaqodoon/shaqodoon.go [hunk 1]
```diff
@@ -1,6 +1,210 @@
...
+	s.AddUpdateStatusRoute(h, "GET", "sent", sentHandler)
+	s.AddUpdateStatusRoute(h, "POST", "sent", sentHandler)
+
+	deliveredHandler := h.buildStatusHandler("delivered")
+	s.AddUpdateStatusRoute(h, "GET", "delivered", deliveredHandler)
+	s.AddUpdateStatusRoute(h, "POST", "delivered", deliveredHandler)
+
+	failedHandler := h.buildStatusHandler("failed")
+	s.AddUpdateStatusRoute(h, "GET", "failed", failedHandler)
+	s.AddUpdateStatusRoute(h, "POST", "failed", failedHandler)
```
Comment (nicpottier) [inline handlers/shaqodoon/shaqodoon.go]: Can we check the log to see how they actually call us and only remap those methods?

File: handlers/shaqodoon/shaqodoon.go [hunk 2]
```diff
@@ -1,6 +1,210 @@
...
+		return nil, err
+	}
+
+	return []courier.Msg{msg}, courier.WriteReceiveSuccess(w, r, msg)
+}
+
+type shaqodoonMessage struct {
+	From   string `name:"from"`
+	Sender string `name:"sender"`
+	Text   string `validate:"required" name:"text"`
```
Comment (nicpottier) [inline handlers/shaqodoon/shaqodoon.go]: Do they actually use both `from` and `sender`? (can check logs to find out)

File: handlers/shaqodoon/shaqodoon.go (hunk 1 above)
Comment (norkans7) [inline handlers/shaqodoon/shaqodoon.go]: logs say no status report is ever sent to us

File: handlers/shaqodoon/shaqodoon.go (hunk 2 above)
Comment (norkans7) [inline handlers/shaqodoon/shaqodoon.go]: logs show from only

**Pull request merged**